app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -26,6 +26,15 @@
             {{#t}}Download Firefox for mobile devices.{{/t}}
           </p>
 
+          <form class="marketing-sms-link">
+            <div class="input-row">
+              <input type="text" id="phone-number" class="phone" />
```

Comment:
Ahha, the form will be presented in addition to the other links, not in lieu of. @ryanfeeley - does that match your expectations?